vms/platformvm/txs/mempool/mempool.go

```diff
@@ -23,37 +22,21 @@ var (
 
 type Mempool interface {
```

Comment:
Seems like we shouldn't have this interface anymore.

Reply:
Removed